source/globalCommands.py

```diff
@@ -2008,18 +2023,20 @@ def script_review_currentSymbol(self,gesture):
 	@script(
 		description=_(
 			# Translators: Input help mode message for toggle speech mode command.
-			"Toggles between the speech modes of off, beep and talk. "
+			"Toggles between the speech modes of off, beep, talk and on-demand. "
 			"When set to off NVDA will not speak anything. "
 			"If beeps then NVDA will simply beep each time it its supposed to speak something. "
 			"If talk then NVDA will just speak normally."
+			"If on-demand then NVDA will only speak when calling commands whose goal is to speak something, but not"
+			" when performing an action such as moving the focus or the cursor."

```

Comment:
Of course, if you accept my former suggestion, this line should go away.

However, more broadly, as it is likely some version of #15806 will be implemented before this leaves alpha, it may be better to shorten this entire long message to something like: "Cycles through the available speech modes."

```suggestion
```